### DISCUSSION TITLE
Websites with the custom domain will display the upper and lower parts when deployed on Cloudflare or netlify.

### DISCUSSION THREAD

**Discussion by zhengbangbo** [vuejs/vitepress#1944] [Q&A] [answered]
Cloudflare provides the second domain name, and the first domain name is my custom domain. They are the **same** deployment, but the display is entirely different.

1. https://handbook.imzbb.cc/ (custom domain, strange display)
2. https://handbook-4fv.pages.dev/ (display correctly)

I searched for those keywords but have no idea: _repetitive_ and _duplicate_. Does anyone have a similar situation?

Repository: https://github.com/zhengbangbo/handbook

**Comment by brc-dd** [comment, marked as answer]
You've auto minify enabled for html. Disable that, it should work.

**Reply by zhengbangbo** [reply]
Thank you very much. After disabling the auto minify of Cloudflare for HTML, this problem has indeed been solved.

https://support.cloudflare.com/hc/en-us/articles/200168196-Using-Cloudflare-Auto-Minify

<img width="1042" alt="image" src="https://user-images.githubusercontent.com/18254135/218969130-ddb82e40-d843-4c30-b115-68d2aace58bf.png">
